orca_detector/vggish_model.py

```diff
@@ -0,0 +1,261 @@
+"""
+VGGish model for Keras. A VGG-like model for audio classification
+
+W251 (Summer 2019) - Spyros Garyfallos, Ram Iyer, Mike Winton
+
+Adapted from https://github.com/DTaoo/VGGish
+"""
+
+import numpy as np
+import sys
+
+from keras.models import Model
+from keras.layers import Flatten, Dense, Input, Conv2D, MaxPooling2D, GlobalAveragePooling2D, GlobalMaxPooling2D
+from keras.engine.topology import get_source_inputs
+from keras import backend as K
+
+# project-specific imports
+import orca_params
+import vggish_params as params
+
+# weight path; when Docker container is run, weights path on the host
+# machine is expected to be mapped to /weights
+WEIGHTS_PATH = '/weights/vggish_audioset_weights_without_fc2.h5'
+WEIGHTS_PATH_TOP = '/weights/vggish_audioset_weights.h5'
+
+class OrcaVGGish(object):    
```

Comment:
why didn't you inherit from VGGish class ?

Reply:
I was somewhat concerned that it wouldn't be totally clear what to override vs. for what to rely on the superclass's functionality.  I don't think we actually need the original any more.  

I will tidy the OrcaVGGish class up a little bit, and when I do, I'll delete the VGGish class.

Reply:
done. please take a look